app/utils/id_generator.py

```
@@ -0,0 +1,163 @@
+        카운터 리셋 (테스트용)
+        
+        Args:
+            entity_type: 엔티티 타입
+            value: 리셋할 값 (기본값: 0)
+        """
+        counter_key = f"{entity_type}_id"
+        
+        lock = FileLock(self.lock_file, timeout=10)
+        
+        with lock:
+            with open(self.counters_file, 'r', encoding='utf-8') as f:
```

Ruff (0.14.11)

Unnecessary mode argument

Remove mode argument

(UP015)